.travis.yml

```diff
@@ -12,6 +12,5 @@ matrix:
 script:
   - mvn -q -B -f jms-light/pom.xml clean verify
   - mvn -q -B -f kafka-connector/pom.xml clean verify
-  - mvn -q -B -f client/pom.xml clean verify -Dmaven.javadoc.skip=true -Dgpg.skip=true -Djava.util.logging.config.file=client/src/test/resources/logging.properties
```

Comment:
should we just keep ` mvn -q -B -f client/pom.xml clean verify -Dmaven.javadoc.skip=true -Dgpg.skip=true -DskipTests=true` ?